src/pages/Download.js

```diff
@@ -65,22 +65,22 @@ const Download = () => {
                         'https://github.com/UPBGE/upbge/releases/download/v0.30/UPBGE-0.30-windows7-x86_64.7z'
                       }
                       target={'_blank'}>
-                      Download UPBGE 0.30.0 (64 bits)
+                      Download UPBGE 0.30.0 (64-bit)
                     </Button>
                   </div>
                   <div>
                     <p>NOTE:</p>
                     <BulletListInside>
-                      <li>64-bits builds do not run at 32-bits systems.</li>
+                      <li>64-bit builds do not run on 32-bit systems</li>
                       <li>
-                        7z extension is opened with{' '}
+                        7z extension can be extracted with{' '}
                         <a
                           href='https://www.7-zip.org/'
                           target={'_blank'}
                           rel='noreferrer'>
                           7zip
                         </a>{' '}
-                        or similar application{' '}
+                        <br/>‎‎&nbsp;&nbsp;&nbsp;&nbsp;or a similar application{' '}
```

Comment:
You're absolutely right, I agree.
Do you know any way to enforce inline sublines if a paragraph comes directly below a `<li>`?

e.g.

* Main `<li>`
Subline 1.1
Subline 1.2

Reply:
I believe the best way to implement that is nesting unordered lists (`ul` > `li` inside another one) then using CSS to apply proper padding.

One useful selector you might want to use is _"ancestor descendants"_, so you can only apply paddings to nested `li` elements with `li li` (select `li`s under any ancestor `li`).

Reply:
Committed a fix that is hopefully better in terms of reliability.
Feel free to suggest or share any ideas/code. I'm mostly new to Tailwind its unique classNames.

Reply:
Looks much better! :) You may want to check your editor settings, by the way, since the last commit seems to have broken the indentation. Either setup the code format settings in your editor to match the convention, or just run `lint-fix` before the commit, which is considered a best practice anyway.

Also, I think it'd be better to replace `{' '}` with `&nbsp;`. While it's not recommended to use `&nbsp;` for specifying a padding, it's a perfectly good usage to express a space between words when a simple space doesn't do the job.

Reply:
The indention is odd, I agree. Before the last mentioned commit, I used Notepad, then switched to [Visual Studio Code Web Edition](https://github.dev/) when I noticed the odd 'diff' and attempted to 'fix' the indention, it looked fine in VSC, but after commit the diff was still odd looking... not sure what is happening but I will definitely try `lint-fix` in the future.

I did not realize `{' '}` was a space... very interesting.